src/api/v1/apiV1Module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { ImportController } from './import/import.controller';
+import { HealthModule } from './health/health.module';
+import { HealthController } from './health/health.controller';
+
+@Module({
+  imports: [HealthModule],
+  controllers: [ImportController, HealthController],
+})
+export class ApiV1Module {}
```

Comment:
This is invalid now, this should be: 
```suggestion
export @Module({
  imports: [HealthModule],
  controllers: [ImportController, HealthController],
})
class ApiV1Module {}
```

Reply:
Whilst I agree it is invalid, it isn't currently supported in either JS or TS natively so won't be able to satisfy the spec